lib/swift/Sources/TProtocol.swift

```diff
@@ -41,7 +41,8 @@ public enum TType: Int32 {
   case set      = 14
   case list     = 15
   case utf8     = 16
-  case utf16    = 17
+  //case utf16    = 17
```

Comment:
is this a breaking change?

Reply:
> utf16

Strings in Thrift ALWAYS go as UTF8 over the wire. Utf16 is not a valid Thrift type, so adding it to TType seems odd.

> Because right now neither utf16 nor uuid are documented types here: https://thrift.apache.org/docs/idl.html

Wrong link, this is release 0.17.0. The one at https://github.com/apache/thrift/blob/master/doc/specs/idl.md is for master.

Reply:
@Jens-G Is the justification for this change satisfactory for you? To be clear, this PR *removes* UTF16 support, and replaces it with UUID to bring Swift in compliance with the spec